src/test/java/org/broadinstitute/hellbender/utils/reference/ReferenceUtilsUnitTest.java

```diff
@@ -52,6 +53,25 @@ public void testLoadFastaDictionaryFromStream() throws IOException {
         }
     }
 
+    @Test
+    public void testLoadFastaDictionaryWithFastaFile() throws IOException {
```

Comment:
The name of this test case will be confusing to anyone not familiar with the original issue -- recommend renaming to something like `testLoadWrongFormatFastaDictionary()`